src/main/kotlin/net/maizegenetics/phgv2/cli/AgcCompress.kt

```diff
@@ -37,23 +43,23 @@ import java.time.LocalDate
  * YYYY-MM-DD is the date the backup was created.
  *
  */
-class AgcCompress : CliktCommand(help = "Create a single AGC compressed file from an input of FASTA files") {
+class AgcCompress : CliktCommand(help = "Create a single AGC compressed file from an input of FASTA files created with the phg annotate-fasta command") {
 
     private val myLogger = LogManager.getLogger(AgcCompress::class.java)
 
     val dbPath by option(help = "Folder name where AGC compressed files will be created.  Should be the same parent folder where tiledb datasets are stored." )
         .default("")
 
 
-    val fastaList by option(help = "File containing full path name for the fasta files, one per line, to compress into a single agc file.  Fastas may be compressed or uncompressed files. Reference fasta should NOT be included.")
+    val fastaList by option(help = "File containing full path name for the annotated fasta files, one per line, to compress into a single agc file.  Fastas may be compressed or uncompressed files. Reference fasta should NOT be included.\nAll fastas (including reference) must be fastas created via the phg annotate-fastas command")
```

Comment:
This says "Reference fasta should NOT be included". But next sentence says "including reference"?